app/pages/feed/feed.scss

```diff
@@ -1,3 +1,3 @@
 .feed {
-  
+  background-color: color($colors, backdrop);
```

Comment:
I made all pages have a background color of grey (saved in a variable named "backdrop" for easy config).

This makes the cards stand out from the background.